app/utils/print/BusinessPersonalRepPrintHelper.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils.print
+
+import javax.inject.Inject
+import models._
+import play.api.i18n.Messages
+import utils.countryOptions.AllCountryOptions
+import viewmodels.{AnswerRow, AnswerSection}
+
+class BusinessPersonalRepPrintHelper @Inject()(countryOptions: AllCountryOptions) {
+
+  def apply(business: BusinessPersonalRep)(implicit messages: Messages): AnswerSection = {
+    val converter = AnswerRowConverter(countryOptions, business.name)
+
+    def name(utr: Option[String]): Seq[AnswerRow] = {
+      utr match {
+        case Some(x) =>
+          Seq(
+            converter.yesNoQuestion(boolean = true, "personalRep.business.ukRegisteredYesNo"),
+            converter.stringQuestion(business.name, "personalRep.business.name"),
+            converter.stringQuestion(x, "personalRep.business.utr")
```

Comment:
Could we not just do an OptionStringQueston that returns an empty Seq() which we flatten rather than duplicate personalRep.business.ukRegisteredYesNo.